<launch>
    <arg name="image_topic" default="/kinect2/hd/image_color"/>
    <arg name="info_topic" default="/kinect2/hd/camera_info"/>
    <arg name="camera_optical_frame" default="kinect2_rgb_optical_frame"/>
    <arg name="image_height" default="1080"/>
    <arg name="image_width" default="1920"/>
    <arg name="marker_size"/>
    <node name="marker_detector" pkg="baxter_kinect_calibration" 
        type="marker_detector" output="screen"
        args="$(arg image_topic) $(arg info_topic) $(arg camera_optical_frame) $(arg image_height) $(arg image_width) $(arg marker_size)"/>
</launch>
